### DISCUSSION TITLE
Welcome to mountebank Discussions!

### DISCUSSION THREAD

**Discussion by bbyars** [bbyars/mountebank#693] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brunns** [comment]
Hi, I'm Simon. I make a lot of use of Mountebank to test microservices, often in Python, to which end I maintain [a Python library which wraps Mountebank](https://pypi.org/project/mbtest/).

**Comment by eljusto** [comment]
Hello, I've implemented a Redis-backed repository https://github.com/eljusto/mountebank-redis-repository. We've started using it in our production CI and it's working well. I'd appreciate any feedback and comments.

**Comment by bbyars** [comment]
Thanks both. I'm a fair bit behind on upkeep, but will add both to the
website for the next release.
-Brandon
